### PR TITLE
Added feature so that questions cannot be edited if survey is open

### PR DESCRIPTION
**Pull request by mack-wallace**

### CODE REVIEW

**Review by DavidHaighton — CHANGES_REQUESTED** [review]
add a test to make sure that this works properly. It'll be just like the test in your other PR

**Comment by mack-wallace** [comment]
> add a test to make sure that this works properly. It'll be just like the test in your other PR

tests added!

**Comment by Shashaank5** [comment]
> add a test to make sure that this works properly. It'll be just like the test in your other PR

@mack-wallace resolve this issue and you should be good to go

File: src/main/resources/templates/EditQuestion.html [hunk 1]
```diff
@@ -66,5 +67,9 @@ <h2 th:text="'Question ID:'+${question.getId()}">Question ID</h2>
     </div>
     <p th:case="*" >Question type not implemented</p>
   </div>
```
Comment (DavidSciola) [inline src/main/resources/templates/EditQuestion.html:69]: Maybe add a button to return home so the user is not stuck on the page

You can use the same button from SurveyFillIssue.html

**Pull request merged**